server/app/__init__.py

```diff
@@ -0,0 +1,9 @@
+from flask import Flask
+from instance.config import App_Configurations
+
+def create_app(Config_Status):
+    app = Flask(__name__)
+    app.config.from_object(App_Configurations[Config_Status])
+    app.config.from_pyfile('../instance/config.py')
+
+    return app
```

Comment:
add a blank line at the end of this file

Reply:
Done as directed